aws-ssm-maintenancewindowtarget/src/test/java/software/amazon/ssm/maintenancewindowtarget/ReadHandlerTest.java

```diff
@@ -0,0 +1,4 @@
+package software.amazon.ssm.maintenancewindowtarget;
```

Comment:
There isn't a ReadHandler for MaintenanceWindowTarget (although I would guess it would be called "GetMaintenaceWindowTarget" if there was). There is a ListHandler but I'm leaving that out because of the lack of a use case. Here is the API for quick access: https://docs.aws.amazon.com/systems-manager/latest/APIReference/API_DescribeMaintenanceWindowTargets.html

Reply:
I think we do need a read handler though as our current CFN supports a read handler for MWTargets. It uses the "DescribeMaintenanceWindowTarget" API. I will ping you the link offline.